<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>620</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 204, 102);</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>500</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir</family>
     </font>
    </property>
    <property name="text">
     <string>Job Description:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="job_description">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>460</y>
      <width>341</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="job_title">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir</family>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Job Title: &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="job_title_edit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>339</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="company">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>260</y>
      <width>171</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir</family>
     </font>
    </property>
    <property name="text">
     <string>Company:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="company_edit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>260</y>
      <width>339</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="location">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>360</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir</family>
     </font>
    </property>
    <property name="text">
     <string>Location:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="location_edit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>360</y>
      <width>339</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="deadline">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>410</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir</family>
     </font>
    </property>
    <property name="text">
     <string>Deadline:</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="application_date">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>410</y>
      <width>339</width>
      <height>31</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2019</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="maximumDateTime">
     <datetime>
      <hour>23</hour>
      <minute>59</minute>
      <second>59</second>
      <year>7999</year>
      <month>10</month>
      <day>30</day>
     </datetime>
    </property>
    <property name="minimumDate">
     <date>
      <year>1950</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
    <property name="displayFormat">
     <string>M/d/yyyy</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="status_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>310</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir</family>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Application status: &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="status">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>310</y>
      <width>339</width>
      <height>31</height>
     </rect>
    </property>
    <property name="modelColumn">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>Applied</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Interview</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hired</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Failed</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../sales tax/logos/logo.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
